app/helpers/offers-count-display.js

```diff
@@ -0,0 +1,22 @@
+import Ember from "ember";
+
+function applyIcon(stateCount, isPriority) {
+  if (!stateCount) {
+    return;
+  }
+  const priorityIcon = `<i class="fa fa-exclamation-triangle"></i> ${stateCount}`;
```

Comment:
Since template code is created, wouldn't a component be more appropriate than a helper? 🤔